File: Pemrograman1/Modul6/Python/PRAK603-2310817310009-MuhammadFauzanAhsani.py
import numpy as np

column1, column2 = map(int, input().split())
if (column1 != column2):
    print("Jumlah tidak sama")
    
else:
    number1 = list(map(int, input().split()))
    number2 = list(map(int, input().split()))

    arr1 = np.array(number1).reshape(1, column1)
    arr2 = np.array(number2).reshape(1, column2)

    for i in range(column1):
        print(arr1[0,i] * arr2[0,i], end=" ")

File: Pemrograman1/Modul6/C/PRAK603-2310817310009-MuhammadFauzanAhsani.c
#include <stdio.h>

int main(){
    int column1, column2;
    scanf("%d %d", &column1, &column2);

    if (column1 != column2){
        printf("Jumlah tidak sama");
    }
    else{
        int arr1[1][column1], arr2[1][column2];

        for (int i = 0; i < column1 ; i++){
            scanf("%d", &arr1[0][i]);
        }
        
        for (int i = 0; i < column2 ; i++){
            scanf("%d", &arr2[0][i]);
        }

        for (int i = 0; i < column1 ; i++){
            printf("%d ", arr1[0][i] * arr2[0][i]);
        }
    }
    return 0;
}